UACloudLibraryServer/PostgreSQLDB.cs

```diff
@@ -356,10 +356,11 @@ public UANodesetResult[] FindNodesets(string[] keywords, int? offset, int? limit
                         .Concat(
                             _dbContext.Metadata
                                 .Where(md => Regex.IsMatch(md.Value, keyword))
-                                .Select(md => md.NodesetId.ToString())
+                                .Select(md => md.NodesetId.ToString(CultureInfo.InvariantCulture))
```

Comment:
This change breaks the FindNodesets (ToString runs in the database which doesn't support cultureinfo). 
PR #144 does not have the issue, or a more targetted fix would be to remove CultureInfo.InvariantCulture.